src/mqueryfront/src/indexFiles/IndexClearQueueButton.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import api from "../api";
+
+class IndexClearQueueButton extends Component {
+    onClick() {
+        api.delete(`/queue/${this.props.ursa_id}`);
+    }
+
+    render() {
+        return (
+            <span
+                data-toggle="tooltip"
+                title={
+                    "This action will clear all unindexed files from this queue"
```

Comment:
```suggestion
                    "This action will remove all files from this queue"
```
I guess (since all files are unindexed already)